docs/release-notes/release-0-8-0.md

```diff
@@ -1,6 +1,16 @@
 # v0.8.0
 
-Release date: May 17, 2024
+Release date: May 18, 2024
+
+## 👍 Highlights
+
+- Flow Engine: which brings continuous aggregation capabilities.
```

Comment:
```suggestion
- [Flow Engine](/user-guide/continuous-aggregation/overview): which brings continuous aggregation capabilities.
```

Reply:
@tisonkun I'm not sure if these links work

Reply:
This should work.

I suggest @WenyXu to create the PR based on the upstream branches so that Cloudflare preview can work.

Reply:
However, I may suggest to use `https://docs.greptime.com/user-guide/...` to keep links to the latest stable version.